sentry-ruby/spec/sentry/scope_spec.rb

```diff
@@ -198,14 +198,16 @@
     end
 
     it "applies event processors to the event" do
-      subject.add_event_processor do |event|
+      subject.add_event_processor do |event, hint|
         event.tags = { processed: true }
+        event.extra = hint
         event
       end
 
-      subject.apply_to_event(event)
+      subject.apply_to_event(event, { foo: "bar" })
 
       expect(event.tags).to eq({ processed: true })
+      expect(event.extra).to eq({ foo: "bar" })
```

Comment:
I guess this is just for testing but the intent of the hint is different ^^

Reply:
yeah it's for testing only. I think hints are for users to distinguish events in custom callback/processors?